docs/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,15 +1,15 @@
 ### What was the problem?
 
-### How did I fix it?
+### How did I solve?
 
-### How to test it?
+### How to manually test it?
 
 ### Review checklist
 
-* The PR resolves #INSERT_ISSUE_NUMBER
-* All new code is covered with unit tests
-* All new code was formatted with Prettier
-* Linting passes
-* Tests pass
-* Commit messages follow the [commit guidelines](CONTRIBUTING.md#git-commit-messages)
-* Documentation has been added/updated
+- The PR resolves #INSERT_ISSUE_NUMBER
```

Comment:
Converting these item marks to checkbox items can increase awareness of the item: `[ ] -`